Makefile

```diff
@@ -12,8 +12,8 @@ build:
 lint-targets := lint-spell lint-git
 
 lint-spell:
-	@$(ECHO) "$(CYAN)*** Runnint cspell...$(OFF)"
-	@npx cspell "src/**"
+	@$(ECHO) "$(CYAN)*** Running cspell...$(OFF)"
+	@npx cspell "src/**" "docs/**" "public/**" "README.md"
```

Comment:
This needs to be kept in sync with:
https://github.com/oasisprotocol/oasis-wallet-ext/blob/5ea38d8fbed2e18fee9a54bc655e34419191c6f3/.github/workflows/ci-lint.yml#L49-L54